app/gui2/src/components/ComponentBrowser/input.ts

```diff
@@ -261,15 +262,18 @@ export function useComponentBrowserInput(
   }
 
   /** Apply given suggested entry to the input. */
-  function applySuggestion(entry: SuggestionEntry) {
+  function applySuggestion(id: SuggestionId) {
+    const entry = suggestionDb.get(id)
+    if (!entry) return
     const { newCode, newCursorPos, requiredImport } = inputAfterApplyingSuggestion(entry)
     code.value = newCode
     selection.value = { start: newCursorPos, end: newCursorPos }
     if (requiredImport) {
-      const [id] = suggestionDb.nameToId.lookup(requiredImport)
-      if (id) {
-        const requiredEntry = suggestionDb.get(id)
+      const [importId] = suggestionDb.nameToId.lookup(requiredImport)
+      if (importId) {
+        const requiredEntry = suggestionDb.get(importId)
         if (requiredEntry) {
+          // Keep in mind, that we need to provide the id of `entry`, not `requiredEntry` here!
```

Comment:
The comment doesn't seem to be relevant anymore.

Reply:
Removed